_platforms/en/designers.md

```diff
@@ -37,27 +37,39 @@ The goal is to provide kits - documents, guides, or software - that can save tim
+
+## UI kit
+
+The UI Kit is a set of visual components already ready to build the interface of a site or an app, following the guidelines for the digital services of the Public Administration. The kit is built with Sketch and defined in an open and collaborative way on GitHub together with the designers community. The UI Kit is the reference point for UI components and patterns. Since 2022, it has been included as part of the design system.
+
+* [Introduction to the UI Kit](https://designers.italia.it/design-system/come-iniziare/per-designer/){:target="_blank"}
+* [UI Kit components and patterns](https://www.figma.com/community/file/1105848677422572920/ui-kit-italia-v3-1-0){:target="_blank"}
 * [GitHub Repository](https://github.com/italia/design-ui-kit){:target="_blank"}
 
-Many Public Administration projects are using the UI Kit to define their own interfaces, including models for [sites of the municipalities](https://github.com/italia/design-comuni-prototipi){:target="_blank"} and [sites of schools](https://github.com/italia/design-scuole-prototipi){:target="_blank"}.
+Many Public Administration projects are using the UI Kit to define their own interfaces, including templates for [sites of the municipalities](https://github.com/italia/design-comuni-prototipi){:target="_blank"} and [sites of schools](https://github.com/italia/design-scuole-prototipi){:target="_blank"}.
 
-For questions and discussions about the UI Kit, on [Developers Italia Slack](https://slack.developers.italia.it/){:target="_blank"}, you can refer to the channel [#design-ui](https://developersitalia.slack.com/messages/C9N62GX8E/){:target="_blank"}.
+For questions and discussions about the UI Kit, on [Developers Italia Slack](https://slack.developers.italia.it/){:target="_blank"}, you can refer to the channel [#design-system-ui-kit](https://developersitalia.slack.com/archives/C9N62GX8E) {:target="_blank"}.
 
 ## Web Development Kit
 
-The [Web Development Kits](https://designers.italia.it/kit/web-development-kit/){:target="_blank"} are a set of tools for the front end development of websites and web applications. They are HTML, CSS and Javascript libraries that allow you to create sites that comply with the Design Guidelines and code the provisions of the UI Kit. As of today, the most stable, updated and mature web development kit is Bootstrap Italia, based on the open-source library Bootstrap 4.
+The [Web Development Kits](https://designers.italia.it/kit/web-development-kit/){:target="_blank"} are a set of tools for the front end development of websites and web applications. They are HTML, CSS and Javascript libraries that allow you to create sites that comply with the Design Guidelines and code the provisions of the UI Kit. As of today, the most stable, updated and mature web development kit is Bootstrap Italia, based on the open-source library Bootstrap 4. Since 2022, they all have been included as part of the design system.
```

Comment:
```suggestion
The [Web Development Kits](https://designers.italia.it/kit/web-development-kit/){:target="_blank"} are a set of tools for the front end development of websites and web applications. They are HTML, CSS and Javascript libraries that allow you to create sites that comply with the Design Guidelines and code the provisions of the UI Kit. As of today, the most stable, updated and mature web development kit is Bootstrap Italia, based on the open-source library Bootstrap 4. They all have been included as part of the design system since 2022. 
```